parse/parse_test.go

```diff
@@ -375,6 +375,16 @@ func TestParseComplexMessage(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:  "trailing whitespace",
```

Comment:
Not needed? The test is duplicated https://github.com/unicode-org/message-format-wg/pull/812/files